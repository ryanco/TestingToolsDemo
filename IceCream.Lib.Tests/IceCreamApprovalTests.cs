using Microsoft.VisualStudio.TestTools.UnitTesting;
using ApprovalTests;

namespace IceCream.Lib.Tests
{
	[TestClass]
	public class IceCreamApprovalTests
	{
		[TestMethod]
		public void IceCreamApprovalTest()
		{
			var iceCream = new IceCreamProduct
				{
					Name = "Rull Guud",
					Cost = "2.50",
					Flavor = "Bumpin Brogrammer"
				};
			Approvals.Verify(iceCream);
		}
	}
}
