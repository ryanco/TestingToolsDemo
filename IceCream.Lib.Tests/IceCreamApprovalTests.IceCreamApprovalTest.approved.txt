Name: Rull Guud Flavor: Bumpin Brogrammer Cost: 2.50